docs/notes/2.23.x.md

```diff
@@ -134,6 +134,8 @@ Support documenting macro constants using `MY_CONSTANT: Annotated[some_type, Doc
 
 Fixed bug where files larger than 512KB were being materialized to a process's sandbox without write permissions if the file was only globbed by `output_directories=(".",)`.
 
+Fixed bug where using `RuleRunner` in plugin tests caused `ValueError` that complains about not finding build root sentinel files. Note that you may have to adjust your tests to account for the new `BUILDROOT` file that `RuleRunner` now injects in the sandbox it creates for each test.
```

Comment:
> Note that you may have to adjust your tests to account for the new `BUILDROOT` file that `RuleRunner` now injects in the sandbox it creates for each test.

What is the practical implication of this? What is the possible adjustment?

Reply:
This might impact tests that use a `**` glob to select files, because that will include the new `BUILDROOT` file, so assertions might fail. I had to adjust some tests like that in the pants repo. In one case, I added an exclude glob to ignore that file. In another case, I listed `BUILDROOT` in the list of expected files.

Reply:
I added the example to the release notes.